indico_toolkit/snapshots/snapshot.py

```diff
@@ -93,7 +93,7 @@ def append(self, snap_to_add: Snapshot):
             snap_to_add (Snapshot): Snapshot to add
         """
         self._assert_key_column_names_match(snap_to_add)
-        self.df = self.df.append(snap_to_add.df, ignore_index=True)
+        self.df = pd.concat([self.df, snap_to_add.df], ignore_index=True)
```

Comment:
Apparently, pandas append is getting deprecated: https://stackoverflow.com/questions/70837397/good-alternative-to-pandas-append-method-now-that-it-is-being-deprecated

Reply:
makes sense